environments/local-docker.yaml

```diff
@@ -13,31 +13,34 @@ azurite:
   service:
     type: LoadBalancer
 
-dapi:
+db:
   enabled: true
-  disabledUrl: "host.docker.internal"
-  replicaCount: 1
-  image:
-    repository: gpitfuturesdevacr.azurecr.io/nhsd/buying-catalogue/document-dapi
-  useLocalImage: false
-  imagePullSecrets: 
-  - name: regcredlocal
   service:
     type: LoadBalancer
-  azureBlobStorage:    
-    containerName: container-1
 
-bapi-db:
+bapi-db-deploy:
+  enabled: true
+  replicaCount: 1
+  imagePullSecrets: 
+  - name: regcredlocal
+  db:
+    includeImport: "TRUE"
+    includePublish: "TRUE"
+    insertTestData: "TRUE"
+  
+dapi:
   enabled: true
   disabledUrl: "host.docker.internal"
   replicaCount: 1
   image:
-    repository: gpitfuturesdevacr.azurecr.io/nhsd/buying-catalogue/debug-db
+    repository: gpitfuturesdevacr.azurecr.io/nhsd/buying-catalogue/document-dapi
```

Comment:
should be able to remove this? The image should be defined in the src/buyingcatalogue/values.yaml file, I think, as it doesn't change between environments.

Reply:
Great point! I'll prune it.